Tamil/Thirukkural_Learning/Learning.md

```diff
@@ -30,3 +30,11 @@ in their face.
 It is the part of the learned to give joy to those whom they meet, and on 
 leaving, to make them think (Oh! when shall we meet them again).
 
+
+****
+இன்மையின் இன்னாது உடைமை முறைசெய்யா 
+மன்னவன் கோற்கீழ்ப் படின்.	
+
+வறுமையின்றி வாழ்ந்தால்கூட அந்த வாழ்க்கை கொடுங்கோல் ஆட்சியின் கீழ் அமைந்துவிட்டால் வறுமைத் துன்பத்தை விட அதிகத் துன்பம் தரக் கூடியது. 	
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.